extension/scripts/features/alcoholnerve/ttAlcoholNerve.js

```diff
@@ -11,7 +11,9 @@
 		{
 			storage: ["settings.pages.items.nerveGains"],
 		},
-		null
+		() => {
+			if (!hasAPIData() || !torndata || !torndata.items) return "No API access.";

```

Comment:
Just having `hasAPIData` is enough. Should there be no `torndata` or `torndata.items`, it would even be better to let it error imo, because that's something that should NEVER happen when someone has entered their api key.